Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1150,31 +1277,36 @@ def search_command(mailbox=None):
     label_ids = [lbl for lbl in args.get('labels-ids', '').split(',') if lbl != '']
     max_results = int(args.get('max-results', 100))
     page_token = args.get('page-token')
-    include_spam_trash = args.get('include-spam-trash', False)
-    has_attachments = args.get('has-attachments')
-    has_attachments = None if has_attachments is None else bool(
-        strtobool(has_attachments))
-    receive_only_accounts = argToBoolean(args.get('show-only-mailboxes', 'false'))
+    include_spam_trash = argToBoolean(args.get('include-spam-trash', False))
+    has_attachments = argToBoolean(args.get('has-attachments', False))
```

Comment:
this looks like a change of behavior: `has attachments` would be None in the past, but now it's always boolean - is this intended?